### PR TITLE
Remove Docker CLI and images parts of the CLI

### LINKED ISSUE #13649: Remove docker cli from Che7
### Description
Che is now deployed on top of k8s platforms (and openshift, etc) and does not work directly on top of Docker, so the CLI which was deprecated needs to be removed before Che7 GA

nightly and latest tag should also be removed from dockerhub

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do ?
Remove Docker CLI and images parts of the CLI for Che 7 GA
Fix #13649
move init files used by keycloak and postgres to their folders

Change-Id: I66dbb2d6df1f3c062e981a6a513c36be8aa1d39a
Signed-off-by: Florent Benoit <fbenoit@redhat.com>

### What issues does this PR fix or reference?
 #13649

### CODE REVIEW

**Comment by benoitf** [comment]
ci-build

**Review by vparfonov — APPROVED** [review]

File: dockerfiles/ip/Dockerfile [hunk 1]
```diff
@@ -1,17 +0,0 @@
-# Copyright (c) 2017 Red Hat, Inc.
```
Comment (riuvshin) [inline dockerfiles/ip/Dockerfile:1]: che-ip is widely used outside CLI, for example https://github.com/eclipse/che/blob/CHE-13649/deploy/openshift/ocp.sh#L243 
so I would propose to keep che-ip

Comment (benoitf) [reply dockerfiles/ip/Dockerfile:1]: @riuvshin I would prefer to use a tagged version there as ocp.sh script is gonna be replaced by chectl as well and fully get rid of che-ip:nightly

Comment (riuvshin) [reply dockerfiles/ip/Dockerfile:1]: so can you please set in this PR che-ip to latest tag

Comment (benoitf) [reply dockerfiles/ip/Dockerfile:1]: sure, it's done

**Review by riuvshin — APPROVED** [review]

**Comment by riuvshin** [comment]
@musienko-maxim Note that after CLI is removed QA team should stop any CLI based deployments testing and appropriate CI jobs will be removed

**Comment by riuvshin** [comment]
@benoitf I've updated corresponding CI jobs so it can be merged now

**Comment by benoitf** [comment]
@riuvshin I'm waiting for someone on QA team and @l0rd to give their +1 and we're good to go thx

**Review by l0rd — APPROVED** [review]

**Review by musienko-maxim — APPROVED** [review]

**Comment by benoitf** [comment]
@riuvshin ok I also need to remove all `:nightly` and `:latest` tags from the dockerhub for these images

**Pull request merged**